src/Akka.Persistence.Sql/Query/SqlReadJournal.cs

```diff
@@ -219,13 +219,14 @@ private Source<EventEnvelope, NotUsed> EventsByPersistenceIdSource(
                             timestamp: r.representation.Timestamp, 
                             tags: r.tags));
 
-        private Source<EventEnvelope, NotUsed> CurrentJournalEvents(long offset, long max, MaxOrderingId latestOrdering)
+        private Source<EventEnvelope, Task<long>> CurrentJournalEvents(long offset, long max, MaxOrderingId latestOrdering)
         {
             if (latestOrdering.Max < offset)
-                return Source.Empty<EventEnvelope>();
+                return Source.Empty<EventEnvelope>().MapMaterializedValue(_ => Task.FromResult(0L));
```

Comment:
If this is how we finally roll (see other comments) I wonder whether or not having our `Task.FromResult(0L)` as a static for older runtimes is a good idea or not, at least for consistency on our side.

(OTOH, I forget if FW runtimes cache `Task.FromResult<long>(0)`... might be overthinking this...)